kolibri/plugins/coach/assets/src/routes/examRoutes.js

```
@@ -47,6 +47,9 @@ export default [
     name: PageNames.EXAM_CREATION_ROOT,
     path: CLASS + QUIZ + '/edit/:sectionIndex',
     component: CreateExamPage,
+    meta: {
```

This silences an error message